presidio-anonymizer/presidio_anonymizer/services/app_entities_convertors.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Dict
+
+from presidio_anonymizer.entities import InvalidParamException
+from presidio_anonymizer.entities.engine import EncryptResult
+from presidio_anonymizer.entities.engine import RecognizerResult, AnonymizeConfig
+
+
+class AppEntitiesConvertor:
+    """Assisting class to convert API json entities to engine entities."""
+
+    @staticmethod
+    def analyzer_results_from_json(data: List[Dict]) -> List['RecognizerResult']:
+        """
+        Go over analyzer results, validate them and convert to List[AnalyzeResult].
+
+        :param data: contains the anonymizers and analyzer_results_json
+        """
+        analyzer_results = []
+        if data is None:
+            raise InvalidParamException(
+                "Invalid input, " "request must contain analyzer results"
+            )
+        for analyzer_result in data:
+            analyzer_result = RecognizerResult.from_json(analyzer_result)
```

Comment:
you can use comprehension
```python
return [RecognizerResult.from_json(analyzer_result) for analyzer_result in data ] 
``` 
and for the other methods below

Reply:
Done.